source/material_model/melt_global.cc

```diff
@@ -198,9 +200,9 @@ namespace aspect
                       if (reaction_rate_out != NULL)
                         {
                           if (c == peridotite_idx && this->get_timestep_number() > 0)
-                            reaction_rate_out->reaction_rates[i][c] = out.reaction_terms[i][c] / this->get_timestep() ;
+                            reaction_rate_out->reaction_rates[i][c] = depletion_change * melting_rate_op_split;
```

Comment:
If the melting_rate_op_split would be a timescale this would look more similar to the old thing just with a changed timescale.